domains/cescdev.json

```
@@ -0,0 +1,9 @@
+{
+"owner": {
+  "username": "GitCesc07",
+"email": "cesar0407197@gmail.com"
+},
+"record": {
+  "CNAME": "cescdev-portafolio.netlify.app/"
```

`CNAME` record must contain domain name only, remove `/`:
```suggestion
  "CNAME": "cescdev-portafolio.netlify.app"
```